node/popn.go

```diff
@@ -784,6 +791,16 @@ func (nd *node) Get(ctx context.Context, args *GetArgs) {
 		sendErr(err)
 		return
 	}
+
+	// if maxppb is not set during get(), use default node's value
+	if args.MaxPPB == 0 {
+		args.MaxPPB = nd.maxPPB
+
+		if args.MaxPPB == 0 {
+			log.Error().Msg("Max Price Per Byte is 0")
+		}
```

Comment:
If we panicked before we wouldn't have to log here

Reply:
maxppb can be "0" if the node would like to only do free retrievals